backend/app.py

```
@@ -1,11 +1,16 @@
 from flask import Flask
 from flask import render_template
 
+from backend.database.db_manager import DatabaseManager
+
 app = Flask(__name__)
+db_manager = DatabaseManager()
+# just separating the routes for better maintenance
+from api import airports, countries, end_results, player
```

Ruff (0.8.2)

`api.airports` imported but unused

Remove unused import

(F401)